installer_scripts/install-yb-voyager

```diff
@@ -99,6 +115,14 @@ ubuntu_main() {
 }
 
 
+macos_main() {
+	macos_install_brew
```

Comment:
can we print a Note to the user saying that its only for postgres

Reply:
Done.